solana/program_v2/packages/client/core/package.json

```diff
@@ -3,8 +3,8 @@
     "version": "1.0.0",
     "description": "gateway-v2-client",
     "license": "MIT",
-    "main": "dist/src/index.js",
-    "typings": "dist/src/index.d.ts",
+    "main": "dist/index.js",
+    "typings": "dist/index.d.ts",
```

Comment:
No source folder here? Haven't looked at the dist in a bit but just wanted to ask

Reply:
since yarn workspace relies to this a problem should show up in the CLI right away.